ami/utils.go

```diff
@@ -38,14 +38,28 @@ func command(action string, id string, v ...interface{}) ([]byte, error) {
 }
 
 func send(ctx context.Context, client Client, action, id string, v interface{}) (Response, error) {
+	if id == "" {
+		id, _ = GetUUID()
```

Comment:
I do not agree to force the inclusion of action id. 
Because, it could be optional and managed outside the library.